starburstlabs/turbolinks-ios
# Fix qa app and ngrok debugging

This PR should have been opened and merged in before https://github.com/starburstlabs/crm-turbolinks-ios/pull/110 got merged in. We use the new Wealthbox framework which conditionally adds an `ngrok` header to requests to an `ngrok` domain. We also allow the webview to be debugged in everything but the production app.

### QA Steps
We already tested the ability to inspect the webview of the QA app build in https://github.com/starburstlabs/crm-turbolinks-ios/pull/110, but we did not test that you are still unable to inspect the webview in the production app.

- Download the regular production Wealthbox app from TestFlight (not the QA app) version 2.7.17
- Ensure you've enabled the Web Inspector in your phone settings (Settings > Safari > Advanced)
![image](https://github.com/starburstlabs/turbolinks-ios/assets/1101860/69d1ba5c-f6d7-4f77-94ca-269ab8de8187)

- Connect your phone to your computer using a cable
- Launch the production iOS app on your phone and log in
- Open Safari on your mac and open the [Develop](https://support.apple.com/lt-lt/guide/safari/sfri20948/17.0/mac/14.0) menu
- [ ] Confirm that you do not see the webview of the production app as an inspectable item
